index.bs

```diff
@@ -2206,9 +2212,9 @@ Integrations {#integrations}
 Feature Policy {#feature-policy}
 --------------
 
-This specification defines a [=policy-controlled feature=] that controls whether the {{Navigator/xr}} attribute is exposed on the {{Navigator}} object.
+This specification defines a [=policy-controlled feature=] that controls whether any {{XRSession}} that makes use of spatial tracking may be returned by {{XR/requestSession()}}, and whether support for session modes that require spatial tracking may be indicated by either {{XR/supportsSession()}} or {{devicechange}} events on the {{Navigator/xr|navigator.xr}} object.
```

Comment:
> makes use of spatial tracking

perhaps this statement should be stronger?  "requires the use of spatial tracking"?

Reply:
Done.